dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/FileAsset.java

```diff
@@ -132,16 +132,16 @@ private Dimension computeFileDimension(final File file) {
 		if (fileDimension == null) {
 			try {
     				return (fileDimension = ImageUtil.getInstance().getDimension(file));
-			} catch (Exception e) {
-				Logger.debug(this,
+			} catch (Throwable e) {
+				Logger.warn(this,
 						"Error computing dimensions for file asset with id: " + getIdentifier(), e);
 			}
 		}
 		return null;
 	}
 
     //Lazy Suppliers are memoized. Meaning that this truly guarantees the computation takes place once.
-    private transient final Lazy<Dimension> lazyComputeDimensions = Lazy.of(() -> computeFileDimension(getFileAsset()));
+    private Lazy<Dimension> lazyComputeDimensions = Lazy.of(() -> computeFileDimension(getFileAsset()));
```

Comment:
Codacy found an issue: [Private field 'lazyComputeDimensions' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5918493)